_episodes/16-parallel.md

```diff
@@ -125,7 +575,7 @@ Create a submission file, requesting more than one task on a single node:
 {{ site.sched.comment }} {{ site.sched.flag.queue }} {{ site.sched.queue.testing }}
 {% include {{ site.snippets }}/parallel/four-tasks.snip %}
 module load python3
-mpirun ./pi.py 1431652028
+mpiexec python pi.py 100000000
```

Comment:
I'm sorry to be picky about this but for this to run on four tasks you are relying on the MPI runtime being smart and picking up information for the resource scheduler. To be clear we should also give the necessary argument to tell it how many tasks to use:
```suggestion
mpiexec -n 4 python pi.py 100000000
```

Reply:
OK,  we've passed deep into "make this a snippet" territory.

Reply:
Understood, and that's mentioned back in the body text about the scheduler handling it how many tasks to start. Might not be a universal rule, but it's more brittle to have the user specify the number of tasks to run, and have to keep that in sync with the `--ntasks` or similar flags in their resource requests.

Reply:
I do think this is best served by a snippet, that leaves people the scope to comment the code as necessary, as well as more straightforward to tweak. 

It's definitely brittle to have to specify the tasks given it has to be done in context, but there are many use case for MPI outside of a scheduler and it would be required there. In any case, I think it's a case of allowing for whatever the instructor is comfortable with, so a snippet will provide that flexibility without us having to make these calls.